src/analysis/convex/strict.lean

```diff
@@ -0,0 +1,413 @@
+/-
+Copyright (c) 2021 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import analysis.convex.basic
+import topology.algebra.mul_action
+import topology.algebra.ordered.basic
+
+/-!
+# Strictly convex sets
+
+This file defines strictly convex sets.
+
+A set is strictly convex if the open segment between any two distinct points lies in its interior.
+
+## TODO
+
+Define strictly convex spaces.
+-/
+
+open set
+open_locale convex pointwise
+
+variables {𝕜 E F β : Type*}
+
+open function set
+open_locale convex
+
+section ordered_semiring
+variables [ordered_semiring 𝕜] [topological_space E] [topological_space F]
+
+section add_comm_monoid
+variables [add_comm_monoid E] [add_comm_monoid F]
+
+section has_scalar
+variables (𝕜) [has_scalar 𝕜 E] [has_scalar 𝕜 F] (s : set E)
+
+/-- Convexity of sets. -/
```

Comment:
Please expand this docstring. It should explain the definition and mention `convex`.